src/features/venuemap/components/MapPinWithCounter/MapPinWithCounter.tsx

```diff
@@ -5,28 +5,33 @@ import { MapPin } from 'ui/svg/icons/MapPin'
 import { getShadow, getSpacing, Typo } from 'ui/theme'
 
 type Props = {
-  count?: number
+  count: number
 }
 
 const NUMBER_LEFT_POSITION = 15
-const NUMBER_TOP_POSITION = -10
-
-export const VenueMapPin: FunctionComponent<Props> = ({ count }) => {
-  const shouldDisplayCounter = count && count > 1
+const NUMBER_TOP_POSITION = -2
+const MAPPIN_SIZE = 32
+const COUNTER_HEIGHT = 16 + 2 // +2 is the top position
+const COUNTER_WIDTH = 27
 
+export const MApPinWithCounter: FunctionComponent<Props> = ({ count }) => {
```

Comment:
MApPinWithCounter
```suggestion
export const MapPinWithCounter: FunctionComponent<Props> = ({ count }) => {
```
🙃

Reply:
😭